setup.py

```diff
@@ -151,7 +151,7 @@
     "ray[tune]",
     "regex!=2019.12.17",
     "requests",
-    "rhoknp>=1.1.0,<1.3.1",
+    "rhoknp>=1.5.0",
```

Comment:
```suggestion
    "rhoknp>=1.5.0,<2.0",
```
let's be a bit restrictive!